demo/src/html/index.html

```diff
@@ -3,22 +3,14 @@
     <head>
         <meta charset="UTF-8" />
         <title><%= htmlWebpackPlugin.options.title %></title>
+        <link rel="preconnect" href="https://fonts.googleapis.com" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+        <link
+            href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
+            rel="stylesheet"
+        />
     </head>
     <body>
-        <header class="container">
```

Comment:
We should not lose this element; modify to `Header` component

Reply:
Similar to `App.tsx`, are we wrapping `Header.tsx` in `<header className="container">`?